compose/config/fields_schema.json

```diff
@@ -37,7 +37,15 @@
 
         "environment": {
```

Comment:
should this use the `list_or_dict` schema? maybe there are differences between them?